packages/core/admin/admin/src/pages/App/index.js

```diff
@@ -49,145 +57,163 @@ function App() {
+        uuid: initData.uuid,
+      }));
+    }
+  }, [initData, isLoadingInit, updateProjectSettings]);
+
+  const { data: telemetryProperties } = useQuery(
+    ['telemetry-properties'],
+    async () => {
+      const {
+        data: { data },
+      } = await get(`/admin/telemetry-properties`, {
+        // NOTE: needed because the interceptors of the fetchClient redirect to /login when receive a
+        // 401 and it would end up in an infinite loop when the user doesn't have a session.
+        validateStatus: (status) => status < 500,
+      });
+
+      return data;
+    },
+    {
+      enabled: !!auth.getToken(),
     }
-  }, [post]);
-
-  useEffect(() => {
-    const getData = async () => {
-      try {
-        const {
-          data: {
-            data: { hasAdmin, uuid, menuLogo, authLogo },
-          },
-        } = await get(`/admin/init`);
-
-        updateProjectSettings({
-          menuLogo: prefixFileUrlWithBackendUrl(menuLogo),
-          authLogo: prefixFileUrlWithBackendUrl(authLogo),
-        });
-
-        if (uuid) {
-          const {
-            data: { data: properties },
-          } = await get(`/admin/telemetry-properties`, {
-            // NOTE: needed because the interceptors of the fetchClient redirect to /login when receive a 401 and it would end up in an infinite loop when the user doesn't have a session.
-            validateStatus: (status) => status < 500,
-          });
-
-          setTelemetryProperties(properties);
-
-          try {
-            const event = 'didInitializeAdministration';
-            await post(
-              'https://analytics.strapi.io/api/v2/track',
-              {
-                // This event is anonymous
-                event,
-                userId: '',
-                deviceId,
-                eventPropeties: {},
-                userProperties: { environment: appInfo.currentEnvironment },
-                groupProperties: { ...properties, projectId: uuid },
-              },
-              {
-                headers: {
-                  'X-Strapi-Event': event,
-                },
-              }
-            );
-          } catch (e) {
-            // Silent.
-          }
-        }
-
-        setState({ isLoading: false, hasAdmin, uuid, deviceId });
-      } catch (err) {
-        toggleNotification({
-          type: 'warning',
-          message: { id: 'app.containers.App.notification.error.init' },
-        });
-      }
-    };
-
-    getData();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [toggleNotification, updateProjectSettings]);
-
-  const setHasAdmin = (hasAdmin) => setState((prev) => ({ ...prev, hasAdmin }));
-
-  const trackingInfo = useMemo(
+  );
+
+  // we can't use useTracking here, because `App` is not wrapped in the tracking provider
+  // context. This shouldn't use `useFetchClient`, because it does not talk to the admin
+  // API
+  React.useEffect(() => {
+    async function trackInitEvent() {
```

Comment:
Note: this is never called, because `appInfo` is always `undefined`. Are we missing a provider?

Reply:
This appears to be broken since https://github.com/strapi/strapi/pull/13813